code/style.css

```diff
@@ -1,58 +1,233 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
 }
 
-p {
+.wrap {
+  padding: 20px;
+  background-color: #c44536;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.slogan {
+  text-align: center;
+  font-size: 45px;
+  padding: 10px;
+  color: white;
+  font-family: 'Ranchers', cursive;
+  background-color: #c44536;
+}
+
+.card-content {
   margin: 1px;
   font-weight: 300;
   background-color: #fff;
 }
 
+.killer-icon {
+  margin-top: 60px;
+  width: 50%;
+  fill: rgba(255,255,255,0.5);
+}
+
+.killer-icon.hide {
+  display: none;
+}
+
+.weapon-icon {
+  margin-top: 60px;
+  width: 60%;
+  fill: rgba(255,255,255,0.3);
+}
+
+.room-icon {
+  margin-top: 70px;
+  width: 70%;
+  fill: rgba(255,255,255,0.3);
+}
+
+.deck-title {
+  background-color: #8d0801;
+  color: white;
+  font-weight: bold;
+  font-size: 20px;
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;
+}
+
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
-  display: flex;
-  justify-content: space-around;
+  margin: 30px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
   align-items: center;
+  justify-items: center;
 }
 
-.killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+.deck {
+  max-width: 250px;
+  height: 433px;
+  background: #ffd046;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: solid 2px black;
+  box-shadow: 10px 10px 5px 0px rgba(0, 0, 0, 0.3) 
```

Comment:
We don't need to use `px` if value is `0` :

```
  box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.3);
```

Moreover, you forgot about semicolon